app/models/manageiq/providers/inventory/persister/builder.rb

```diff
@@ -237,6 +245,14 @@ def skip_model_class(skip = true)
         @options[:without_model_class] = skip
       end
 
+      def skip_sti
```

Comment:
should this be named `without_sti` for consistency with the other method named `with_sti`?  Also, should these be allowed to take a code block so that one can do something like
```ruby
with_sti { do something interesting }
without_sti { do another interesting thing }
```

Reply:
Actually `skip_sti` is the consistent one as there is already `skip_auto_inventory_attributes` and `skip_model_class`

This persister/builder class is ripe for a lot of cleanup, the way it handles these options are definitely on the list.

Reply:
I can go back to `skip_sti(skip = true)` which would be consistent with the rest of these, I was trying to address the rubocop